avocado/utils/tb.py

```diff
@@ -0,0 +1,39 @@
+"""
+Traceback standard module plus some additional APIs.
```

Comment:
Why not calling it traceback? tb is too short. After that, you'll import from avocado and can make tb an alias, like: `from avocado.utils import traceback as tb`.